srcpkgs/python3-google-api-core/template

```diff
@@ -0,0 +1,16 @@
+# Template file for 'python3-google-api-core'
+pkgname=python3-google-api-core
+version=1.17.0
+revision=1
+archs=noarch
+wrksrc="${pkgname#*-}-${version}"
+build_style=python3-module
+hostmakedepends="python3-setuptools"
+depends="python3-googleapis-common-protos python3-protobuf python3-google-auth
+ python3-requests python3-six python3-pytz"
```

Comment:
I applied this on top of this change:

```diff
diff --git a/srcpkgs/python3-google-api-core/template b/srcpkgs/python3-google-api-core/template
index 411f352970..1e7bb457e0 100644
--- a/srcpkgs/python3-google-api-core/template
+++ b/srcpkgs/python3-google-api-core/template
@@ -8,6 +8,7 @@ build_style=python3-module
 hostmakedepends="python3-setuptools"
 depends="python3-googleapis-common-protos python3-protobuf python3-google-auth
  python3-requests python3-six python3-pytz"
+checkdepends="$depends python3-mock python3-pytest"
 short_desc="Google API client core library for Python3"
 maintainer="Peter Bui <pbui@github.bx612.space>"
 license="Apache-2.0"
```

I run into failure with:
```
======================================================================
ERROR: tests.unit.test_operation (unittest.loader._FailedTest)
----------------------------------------------------------------------
ImportError: Failed to import test module: tests.unit.test_operation
Traceback (most recent call last):
  File "/usr/lib/python3.8/unittest/loader.py", line 436, in _find_test_path
    module = self._get_module_from_name(name)
  File "/usr/lib/python3.8/unittest/loader.py", line 377, in _get_module_from_name
    __import__(name)
  File "/builddir/google-api-core-1.17.0/tests/unit/test_operation.py", line 20, in <module>
    from google.api_core import operations_v1
  File "/builddir/google-api-core-1.17.0/google/api_core/operations_v1/__init__.py", line 17, in <module>
    from google.api_core.operations_v1.operations_client import OperationsClient
  File "/builddir/google-api-core-1.17.0/google/api_core/operations_v1/operations_client.py", line 40, in <module>
    from google.api_core import gapic_v1
  File "/builddir/google-api-core-1.17.0/google/api_core/gapic_v1/__init__.py", line 16, in <module>
    from google.api_core.gapic_v1 import config
  File "/builddir/google-api-core-1.17.0/google/api_core/gapic_v1/config.py", line 23, in <module>
    import grpc
ModuleNotFoundError: No module named 'grpc'

```

I think we'll need `python3-grpc`, no?

Reply:
Ah, I see, it's in extra.